dll/win32/browseui/desktopipc.cpp

```diff
@@ -347,9 +347,11 @@ static HRESULT ExplorerMessageLoop(IEThreadParamBlock * parameters)
     MSG Msg;
     BOOL Ret;
 
+#ifndef _M_AMD64 // FIXME: BROKEN!
     // Tell the thread ref we are using it.
```

Comment:
Sync adding `#ifndef` before or after comments.
Should it be `_WIN64`, like the 2 others?